test/integration/etcd/data.go

```diff
@@ -348,14 +340,6 @@ func GetEtcdStorageDataForNamespace(namespace string) map[schema.GroupVersionRes
 		},
 		// --
 
-		// k8s.io/kubernetes/pkg/apis/storage/v1beta1
-		gvr("storage.k8s.io", "v1beta1", "csistoragecapacities"): {
```

Comment:
this was removed in 1.27 and can be re-enabled for one release past that, so this should hold until 1.29